src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -1,48 +1,48 @@
-<!-- select2 from ajax -->
 @php
     $connected_entity = new $field['model'];
+
     $connected_entity_key_name = $connected_entity->getKeyName();
     $old_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? false;
+
+    $response_entity = isset($field['response_entity']) ? $field['response_entity'] : $crud->hasOperationSetting('ajaxEntities') ? array_has($crud->getOperationSetting('ajaxEntities'), $field['entity']) ? $field['entity'] : array_key_first($crud->getOperationSetting('ajaxEntities')) : '';
```

Comment:
I would rather let the user change the entity using the regular ```data_source``` attribute instead of adding a new attribute. Since it's the only place we're using ```response_entity```, I would remove that entirely, in favour of something like:

```php
$field['data_source'] = $field['data_source'] ?? url($crud->route.'/fetch/'.$field['entity']);
```

If the user would like to change the URL from the field definition array, they can do
```php
'data_source' => url($this->crud->route.'/fetch/smth-custom'),
```

This has two benefits:
- no new syntax;
- you're able to send the request anywhere you'd like, not just inside the current controller, inside the fetch route segment;

Reply:
Added the same comment to the latest review.